src/OrchardCore.Modules/OrchardCore.Queries/Sql/GraphQL/SqlQueryFieldTypeProvider.cs

```diff
@@ -22,12 +23,15 @@ public sealed class SqlQueryFieldTypeProvider : ISchemaBuilder
 {
     private readonly IHttpContextAccessor _httpContextAccessor;
     private readonly ILogger _logger;
+    internal readonly IStringLocalizer S;
 
     public SqlQueryFieldTypeProvider(
         IHttpContextAccessor httpContextAccessor,
+        IStringLocalizer<SqlQueryFieldTypeProvider> localizer,
```

Comment:
for consistency I would name it stringLocalizer

Reply:
It's named differently all over the place and I would prefer the shorter name. But I changed it to your preference.